test/matrix_spec_coverage.py

```diff
@@ -0,0 +1,94 @@
+import sys
+import re
+import yaml
+from responses import RequestsMock
+
+INTERPOLATIONS = [
+    ("%CLIENT_MAJOR_VERSION%", "r0")
```

Comment:
As discussed the spec only lists `%CLIENT_MAJOR_VERSION%` for all it's docs, potentially this is fail-prone where `api/v1` and `api/v2` actually differ in Synapse, probably needs to have an issue opened on matrix-doc.